comps/spec_decode/text-generation/vllm/docker/Dockerfile

```diff
@@ -0,0 +1,226 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Need to refine follow the new code structure, there should be no `docker` folder.   
So this file should named as `Dockerfile.nvidia_gpu`.